### PR TITLE
Update fields for rebrand

### PR DESCRIPTION
**Pull request by jmcolella**

### CODE REVIEW

**Comment by jmcolella** [comment]
@drewdrewthis @sunnymis CR please

[CH](https://app.clubhouse.io/policygenius/story/8820/user-should-see-updated-navigator-components)

- Updates `TextField` and `SelectField` for rebrand
- Adds `FieldGroup` to handle a group of multiple select/text fields

File: src/molecules/formfields/DateField/index.js [hunk 1]
```diff
@@ -43,7 +45,18 @@ function DateField(props) {
         }}
         onFocus={input && input.onFocus}
       >
-        <label htmlFor='date' className={styles['label']}>{label}</label>
+        <div className={styles['label']}>
+          <label htmlFor='date'>{label}</label>
+          {
+            subLabel &&
+              <Text
+                type={10}
```
Comment (sunnymis) [inline src/molecules/formfields/DateField/index.js]: We are deprecating `type`. Use `size` instead

File: src/assets/stylesheets/base_styles/_colors.scss [hunk 2]
```diff
@@ -1,4 +1,25 @@
...
+  primary-2: #ffffff,
+  primary-3: #2b2727,
+
+  /* Assorted accent colors */
+  secondary-1: #7023ff,
+  secondary-2: #00aeff,
+  secondary-3: #ff3ab9,
+  secondary-4: #00cc00,
+  secondary-5: #ffc700,
+);
```
Comment (sunnymis) [inline src/assets/stylesheets/base_styles/_colors.scss]: Sweet

File: src/molecules/formfields/FieldGroup/index.js [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+          <div
+            key={`field--label-${idx}`}
+            className={styles['label']}
+          >
+            <label>{label.header}</label>
+            {
+              label.subLabel &&
+                <Text
+                  type={10}
+                  font='b'
```
Comment (sunnymis) [inline src/molecules/formfields/FieldGroup/index.js]: Use `size`

File: src/molecules/formfields/SelectField/index.js [hunk 4]
```diff
@@ -54,6 +68,12 @@ function SelectField( props ) {
           <label className={styles['label']} htmlFor={forProp}>
             { label }
             { renderToolTip(tooltip) }
+            {
+              subLabel &&
+                <Text type={10} font='b'>
```
Comment (sunnymis) [inline src/molecules/formfields/SelectField/index.js]: ☝️

File: src/molecules/formfields/SelectField/index.js [hunk 5]
```diff
@@ -120,6 +145,14 @@ SelectField.propTypes = {
   })),
 
   /**
+   * Optional text displayed directly underneation label
+   */
```
Comment (sunnymis) [inline src/molecules/formfields/SelectField/index.js]: `underneation`?

**Pull request merged**